jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet

```diff
@@ -236,13 +236,22 @@ function(params) {
         '--tls-cipher-suites=' + std.join(',', pa._config.tlsCipherSuites),
       ],
       resources: pa._config.resources,
+      startupProbe: {
+        httpGet: {
+          path: '/livez',
+          port: 'https',
+          scheme: 'HTTPS',
+        },
+        initialDelaySeconds: 30,
+        periodSeconds: 10,
+        failureThreshold: 180,
```

Comment:
10 x 180 = 30 minutes is extremely generous. Did you have concrete numbers for how long it may take for the pod to be live?

Reply:
Yeah this was a mistake I wanted to put 18, which will give the adapter 3min to boot. On some of the logs from the issue I saw it was taking 2min to boot so I've set it to 3 minutes